### PR TITLE
feat(utils/remove_transparency): remove transparency of pngs

### LINKED ISSUE #64: TV Art: remove transparency from PNG images
Currently, the ffmpeg video creator script respects transparency, so if a PNG with transparency is added, you can see the previous image behind it.

What we want is for all PNGs to have a black background to replace transparency.

So... as part of the media_integrity check, find a way to remove the transparency of png backgrounds and replace with black.

### PR DESCRIPTION
**Pull request by punctuations**
added remove_transparency func to utils, used in image_verify.

Closes #64

### CODE REVIEW

**Comment by punctuations** [comment]
closes #64

File: scripts/_utils/utils.py [hunk 1]
```diff
@@ -129,8 +129,23 @@ def process_svg(svg_url) -> Tuple[bool, Union[str, None], bool]:
         return True, '/tmp/verified-svg.png', True
     else:
         return False, svg_url, False
-        
-        
+
+
+def remove_transparency(im, file_url) -> Tuple[bool, Union[str, None], bool]:
```
Comment (tylerecouture) [inline scripts/_utils/utils.py:134]: What is `im`?  Either use unambiguous variable names or document these parameters in the docstring. Is it an Image object? a filepath? a url?  Does this method expect only png images?  if yes, then needs to be in docstring.  If not, then document how it handles other media types.

File: scripts/_utils/utils.py [hunk 2]
```diff
@@ -152,6 +167,8 @@ def verify_image_integrity(file_url: str, mime: str, local: bool) -> Tuple[bool,
 
     if mime == 'image/svg+xml':
         return process_svg(file_url)
+    elif mime == 'image/png':
+        return remove_transparency(im, file_url)
     elif mime == 'image/gif':
```
Comment (tylerecouture) [inline scripts/_utils/utils.py:172]: Don't we also want svg and gifs to have transparency removed?  Seems like they should both go through the png process AFTER they are svg or gif processed.

**Review by tylerecouture — CHANGES_REQUESTED** [review]

File: scripts/_utils/utils.py (hunk 1 above)
Comment (punctuations) [reply scripts/_utils/utils.py:134]: it is in the docstring

> Removes transparent background from png to opt for a black background

File: scripts/_utils/utils.py (hunk 2 above)
Comment (punctuations) [reply scripts/_utils/utils.py:172]: yes; however, I was going to separate it into two different PRs as it's two different issues, would you like me to combine them?

File: scripts/_utils/utils.py (hunk 1 above)
Comment (punctuations) [reply scripts/_utils/utils.py]: removed ambiguity per [2291e85](https://github.com/timberline-secondary/hackerspace-scripts-2/pull/72/commits/2291e853d1c3983bd9aa1c3cdd4b67edbd2540d4)

Comment (tylerecouture) [reply scripts/_utils/utils.py]: @punctuations I still don't know what datatype `image` parameter is.  There's no type hint.  docstring jsut says it's a png but png isn't a datatype, is is a string url? a string path? a File object? an PIL Image object?  Ditto the others.

**Pull request merged**